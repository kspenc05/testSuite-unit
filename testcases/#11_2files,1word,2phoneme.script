#!/bin/sh
./cmulookup cmudict-0.7b input/0words.txt input/2phonemes_1word.txt