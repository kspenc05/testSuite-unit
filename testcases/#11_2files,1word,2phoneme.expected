=====
T AH0 M EY1 T OW2
T AH0 M AA1 T OW2
=====
DONE! (1 words)
